pom.xml

```diff
@@ -1,24 +1,26 @@
-<?xml version="1.0"?>
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>3.50</version>
+		<version>4.66</version>
 		<relativePath />
 	</parent>
 	<groupId>org.jvnet.hudson.plugins</groupId>
 	<artifactId>monitoring</artifactId>
 	<packaging>hpi</packaging>
-	<version>1.95.0-SNAPSHOT</version>
+	<version>${revision}${changelist}</version>
```

Comment:
Are you sure that `mvn release:prepare release:perform mvn release:prepare release:perform -Dtag=... -DreleaseVersion=... -DdevelopmentVersion=...` will respect that ? or is there another way to release after that ?

Reply:
Yes, I am sure. I have done this literally hundreds of times.

Reply:
ah I understand, that's incremental and automated release. Except that automated release and automated release notes are not compatible with my release process. And I would have preferred that you explain the goals first.

Reply:
This repository does not use automated release, nor does this PR add automated release to this repository.

For non-automated release, there is only one release process that is supported, which is the one documented here: https://www.jenkins.io/doc/developer/publishing/releasing-manually/